package.json

```diff
@@ -52,9 +52,9 @@
     "test.watch": "jest --watch",
     "test.watch-all": "jest --watchAll --coverage",
     "tsc": "tsc --incremental",
-    "tsc.prod": "tsc",
+    "tsc.prod": "tsc --build --force",
```

Comment:
the `--force` flag rebuilds all projects even if `tsc` already has an up-to-date version hanging around